Samples/PowerShell/WindowsStartupShutdownScripts/AddNodeToOrionOnStartup.ps1

```diff
@@ -0,0 +1,110 @@
+# This sample script demonstrates how to add a new node using CRUD operations.
+#
+# Please update the hostname and credential setup to match your configuration, and
+# information about the node you would like to add for monitoring.
+
+#Install Nuget and SwisPowerShell modules
+#On some environments we encountered an issue with executing below command, in that case try adding this command before: [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12, it will set tls version to 1.2
+Find-PackageProvider -Name "Nuget"  -RequiredVersion "2.8.5.208" | Install-PackageProvider -Force
+
+#The below line can be deleted if SwisPowerShell is already installed
+Install-Module -Name SwisPowerShell -Force 
+Import-Module SwisPowerShell -Force
+
+#Decide if you want monitor machine by Agent
+$deployAgent = $true
+
+# Connect to SWIS
+# Orion Host IP
+$hostname = "10.150.16.130"
+# Orion username
+$username = "admin"
+# Orion password
+$password = ConvertTo-SecureString -String "123" -AsPlainText -Force
```

Comment:
If this script is added to the Local Group Policy Editor, won't users of the systems covered by that policy be able to read the script and find the credentials to connect to the Orion server?  I don't believe this is a safe approach.